### PR TITLE
[WIP] Web browser

### PR DESCRIPTION
**Pull request by jkimbo**
kapture 2018-08-26 at 11 12 59

### CODE REVIEW

File: frontend/src/pages/Web/components/WebBrowser.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          backgroundColor: '#fff',
+          padding: '10px',
+          borderRadius: 4,
+          flex: 1,
+        })}
+      >
+        <span className={css({ color: isSecure ? 'green' : 'red' })}>
+          {isSecure ? 'https' : 'http'}
+        </span>
+        <span>://{url}</span>
```
Comment (bendiggle) [inline frontend/src/pages/Web/components/WebBrowser.js]: Is this meant to be commented out?

**Comment by jkimbo** [comment]
Move the Web Browser to the root of the app and trigger opening and closing through redux so that it can be opened from anywhere. Also wrap the webpages in an in memory react router instance so that the webpages can act as if they are a in an actual browser window.

File: frontend/src/reducers/ui/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+// @flow
+import { createSelector } from 'reselect';
+import produce from 'immer';
+
+import websites from '../../components/WebBrowser/websites';
+
+// eslint-disable-next-line no-undef
+type WebPage = $Keys<typeof websites>;
```
Comment (DanielGeoffreyHodgkins) [inline frontend/src/reducers/ui/index.js]: what does this do

**Comment by webbo89** [comment]
Can this go in!

**Pull request merged**